src/appengine/libs/issue_management/issue_tracker_utils.py

```diff
@@ -22,6 +22,14 @@
 from metrics import logs
 
 _ISSUE_TRACKER_CACHE_CAPACITY = 8
+_ISSUE_TRACKER_CONSTRUCTORS = {
+    'monorail': monorail.get_issue_tracker,
+}
+
+
+def register_issue_tracker(tracker_type, constructor):
+  """Register an issue tracker implementation."""
+  _ISSUE_TRACKER_CONSTRUCTORS[tracker_type] = constructor
```

Comment:
nit: Should we add something to skip registration if exists already ?

Reply:
made this raise an exception. should only be registered once.